client/lib/plans/constants.js

```diff
@@ -107,9 +107,11 @@ export const plansList = {
 		getDescription: () => i18n.translate( 'Your own domain name, powerful customization options, and' +
 			' lots of space for audio and video.' ),
 		getDescriptionWithGoogleVouchers: () => i18n.translate( 'Your own domain name, powerful customization options,' +
-			' lots of space for audio and video, and $100 advertising credit.' ),
+			' lots of space for audio and video, and $100 advertising voucher.' ),
 		getDescriptionWithWordAdsInstantActivation: () => i18n.translate( 'Your own domain name, powerful' +
 			' customization options, easy monetization with WordAds and lots of space for audio and video.' ),
+		getDescriptionWithWordAdsAndGoogleVouchers: () => i18n.translate( 'Your own domain name, powerful' +
```

Comment:
Hmm, I know the prop name is quite long already but the `WordAds` part might be confusing with `WordAdsCredit` which is something different than `WordAdsInstantActivation`. Dunno if we should rename this to `getDescriptionWithWordAdsInstantActivationAndGoogleVouchers` (which is preeeety long) or hope no one will confuse it with `WordAdsCredit`.